frontend/src/components/Field/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  ChangeEvent,
+  cloneElement,
+  isValidElement,
+  ReactNode,
+  useRef,
+} from 'react'
+import { BiCheck } from 'react-icons/bi'
+import {
+  Box,
+  Checkbox as ChakraCheckbox,
+  CheckboxProps,
+  ComponentWithAs,
+  Flex,
+  forwardRef,
+  Icon,
+  useMergeRefs,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+type CheckboxComponent = ComponentWithAs<'input', CheckboxProps> & {
+  Others: typeof CheckboxOthers
+}
+
+// forward ref to be consistent with chakra's checkbox
+export const Checkbox = forwardRef<CheckboxProps, 'input'>((props, ref) => {
+  return <ChakraCheckbox ref={ref} {...props} icon={<Icon as={BiCheck} />} />
```

Comment:
Changing the icon may have affected the look and size and animation of the check.
Chakra has an [animated checkbox component](https://github.com/chakra-ui/chakra-ui/blob/main/packages/checkbox/src/checkbox-icon.tsx), we might want to do the same for our replaced icon. 
(However may not be worth it, I think the Chakra default checkbox looks fine as is. Double check with pearly to see if we really need to use `BiCheck`?) 

If not I think we can revert instead of spending time to get the check to look and feel as before (with BiCheck)